### DISCUSSION TITLE
doris使用hive的udf

### DISCUSSION THREAD

**Discussion by denganming** [apache/doris#10652] [A - General / Q&A]
官网说doris可以使用hive的函数，没找到例子，求帮助。
我写的语句
```
CREATE FUNCTION string_kv_2_json2(String,String,String,boolean) RETURNS String PROPERTIES (
    "file"="file:///opt/fe/lib/doris-udf-1.0-SNAPSHOT.jar",
    "symbol"="udf.StringKV2JsonUdf",
    "type"="HIVE"
);
```
报错：No 'object_file' in properties
但是我看object_file是需要http的形式，能否有本地文件的方式

**Comment by hf200012** [comment]
暂时不支持java udf函数，这个要到1.2版本才支持java udf

**Reply by denganming** [reply]
master支持吗，我直接编译master分支使用呢

**Reply by zhangstar333** [reply]
master 支持，https://doris.apache.org/zh-CN/docs/ecosystem/udf/java-user-defined-function.html#类型对应关系

**Reply by KEN-LJQ** [reply]
We encount some problem with current master version, using demo code from https://doris.apache.org/zh-CN/docs/ecosystem/udf/java-user-defined-function/. And I can registry java udf funcion, but when invoke function, it show exception `Unable to load /usr/local/doris-1.2.0/be/lib/udf/77/13005.8e07ec4cae0c7e167b402f773b49f817.so`.
Have I do something wrong?

**Reply by zhangstar333** [reply]
could you give a WeChat or email to discuss the detail?
